client/my-sites/pages/page.jsx

```diff
@@ -378,8 +394,11 @@ const Page = React.createClass( {
 
 export default connect(
 	( state, props ) => {
+		const frontPage = getSiteFrontPage( state, props.page.site_ID );
 		return {
-			isFrontPage: isFrontPage( state, props.page.site_ID, props.page.ID )
+			hasStaticFrontPage: !! ( frontPage && frontPage !== 0 ),
```

Comment:
Question, should this be a proper selector so we don't have various components implementing this logic? Pretty sure I think so.